dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/events/EventsTests.java

```diff
@@ -174,6 +180,48 @@ public void shouldImportToRepeatableStage( Boolean repeatableStage )
         }
     }
 
+    @ParameterizedTest
+    @ValueSource( strings = { "true", "false" } )
+    public void shouldImportEventWithOrgUnitDifferentFromEnrollmentOrgUnit( Boolean repeatableStage )
+        throws Exception
+    {
+        String programId = Constants.TRACKER_PROGRAM_ID;
```

Comment:
May be can add 'get' on the test name to be explicit. Otherwise the test is about importing an event with orgunit different from enrollment orgunit and test by checking explicitly by using different orgunits - hence the ou parameters.

Reply:
ok, I still see _Boolean repeatableStage_  is not used in the test. Maybe it is a copy and paste from another test?

Reply:
thanks. yes, removed that.

Reply:
i think you can safely remove:

```
    @ParameterizedTest
    @ValueSource( strings = { "true", "false" } )

```
and add @Test

Reply:
yes, done that.